logger/config.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  config.swift
+//  Test
+//
+//  Created by Daniel Brain on 10/12/20.
+//
+
+import Foundation
+
+// Time to wait/debounce after flush() calls
+let FLUSH_DEBOUNCE_TIME = 0.5
+
+// Default log level to print in console
+let DEFAULT_LOGLEVEL: LogLevel = .info
+
+// Default interval after which to auto-flush logs
+let DEFAULT_FLUSH_INTERVAL: Double = 60.0
+
+// Maximum number of logs to keep in buffer between flushes
+let MAX_BUFFERED_LOGS = 50
+
+// Log level priority
+let LOG_LEVEL_PRIORITY = [
+    LogLevel.error,
+    LogLevel.warn,
+    LogLevel.info,
+    LogLevel.debug
+]
```

Comment:
I would recommend wrapping this in a struct with static constants to organize it a bit and improve readability. Config values would then be accessible via `Config.FLUSH_DEBOUNCE_TIME `
```suggestion
struct Config {
    // Time to wait/debounce after flush() calls
    static let FLUSH_DEBOUNCE_TIME = 0.5

    // Default log level to print in console
    static let DEFAULT_LOGLEVEL: LogLevel = .info

    // Default interval after which to auto-flush logs
    static let DEFAULT_FLUSH_INTERVAL: Double = 60.0

    // Maximum number of logs to keep in buffer between flushes
    static let MAX_BUFFERED_LOGS = 50

    // Log level priority
    static let LOG_LEVEL_PRIORITY = [
        LogLevel.error,
        LogLevel.warn,
        LogLevel.info,
        LogLevel.debug
    ]
}
```

Reply:
Good call!